lib/filterable/filter.rb

```diff
@@ -50,8 +52,10 @@ def validation(_)
 
     def apply!(collection, value:, active_sorts_hash:, params: {})
       if type == :scope
-        if value.present?
+        if value.present? && scope_params.empty?
           collection.public_send(internal_name, parameter(value))
+        elsif value.present? && scope_params.any?
```

Comment:
Switched this up so we have the `mapped_scope_params` return a hash. It would be cool in the future to either pull `value` into that hash so we can tidy up that `if/else`, or work some magic to get this working with keyword args, instead of handing back a hash.